<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Live Control</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Posi&amp;tion:</string>
           </property>
           <property name="buddy">
            <cstring>posBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="posBox">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="prefix">
            <string> </string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="posSlider">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playBtn">
           <property name="text">
            <string>&amp;Play</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>A&amp;ccel:</string>
           </property>
           <property name="buddy">
            <cstring>accelBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="accelBox">
           <property name="specialValueText">
            <string/>
           </property>
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>-500</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="accelSlider">
           <property name="minimum">
            <number>-500</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="accelResetBtn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_Text_File"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionSetup_Outputs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSetup_Outputs">
   <property name="text">
    <string>Setup Ou&amp;tput</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionOpen_Text_File">
   <property name="text">
    <string>&amp;Open File ...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>posBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>posSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>posSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>posBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accelBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>accelSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>accelBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
